app/models/merge_request.rb

```diff
@@ -31,4 +31,12 @@ def organisation_name_uniqueness
   def absorbing_organisation_name
     absorbing_organisation&.name || ""
   end
+
+  def requesting_user
+    User.find(requester_id) if requester_id
+  end
+
+  def dpo_user
+    absorbing_organisation.users.filter_by_active.find_by(is_dpo: true) if absorbing_organisation
```

Comment:
`absorbing_organisation.data_protection_officers.filter_by_active.first` ? Also, what would happen if organisation doesn't have any DPOs?

Reply:
That works.

If the org doesn't have any then the notification at the top will ask them to assign one. Screenshot in the description.